bitnami/thanos/templates/_helpers.tpl

```diff
@@ -311,7 +313,7 @@ Return true if a configmap object should be created
 Return true if a configmap object should be created
 */}}
 {{- define "thanos.storegateway.createConfigmap" -}}
-{{- if and .Values.storegateway.config (not .Values.storegateway.existingConfigmap) }}
+{{- if and (or (or .Values.storegateway.config .Values.indexCacheConfig) .Values.bucketCacheConfig) (not .Values.storegateway.existingConfigmap) }}
```

Comment:
There is no need to nest two OR statements.

This would act the same:
`(or .Values.storegateway.config .Values.indexCacheConfig .Values.bucketCacheConfig)`

Reply:
Cool, didn't know this was possible :) Thx for the tip!